python/setup.py

```diff
@@ -14,6 +14,7 @@
 import sys
 import setuptools
 import os
+import ctypes
 
 # libin to change to python-dotenv or whatever's simplest:
```

Comment:
Can you remove obsolete commnets and code like this - thanks!

Reply:
Hi Libin, if it wasn't clear I mean the # comments and code below this line. Thanks, Alex